package.json

```diff
@@ -92,6 +92,7 @@
     "ini": "1.3.6",
     "json2csv": "3.11.5",
     "jwt-decode": "2.2.0",
+    "keytar": "5.4.0",
```

Comment:
Where is the `keytar` dependency coming from?  Happened to @markowsiak the other day as well.

Reply:
Hmm. It's an optional dependency already https://github.com/Automattic/vip/blob/master/package.json#L108

It seems that `npm` is automatically moving it into dependencies for some reason?

Reply:
I didn't see explicitly why either.
I just thought if npm was going to move it to be non-optional, then it would remove it from the optional dep section as well 🤷  (unless to preserve some backwards compatibility with other versions of npm?)